packages/core/src/metrics/index.ts

```diff
@@ -0,0 +1,43 @@
+import type { DsnComponents, DynamicSamplingContext, SdkMetadata, StatsdEnvelope, StatsdItem } from '@sentry/types';
+import { createEnvelope, dropUndefinedKeys, dsnToString } from '@sentry/utils';
+
+/**
+ * Create envelope from a metric aggregate.
+ */
+export function createMetricEnvelope(
+  // TODO(abhi): Add type for this
+  metricAggregate: string,
+  dynamicSamplingContext?: Partial<DynamicSamplingContext>,
+  metadata?: SdkMetadata,
+  tunnel?: string,
+  dsn?: DsnComponents,
+): StatsdEnvelope {
+  const headers: StatsdEnvelope[0] = {
+    sent_at: new Date().toISOString(),
+  };
+
+  if (metadata && metadata.sdk) {
```

Comment:
Why don't we use optional chaining in Typescript code?

Reply:
https://blog.sentry.io/performance-impact-of-generated-javascript/#removing-optional-chaining

Reply:
We emit es6 compat bundles - and this metrics code has to be isomorphic (it's why it's in `@sentry/core`)